src/Controls/src/Core/RadioButton.cs

```diff
@@ -542,12 +543,12 @@ static View BuildDefaultTemplate()
 
 			VisualState checkedVisualState = new VisualState() { Name = CheckedVisualState };
 			checkedVisualState.Setters.Add(new Setter() { Property = OpacityProperty, TargetName = CheckedIndicator, Value = 1 });
-			checkedVisualState.Setters.Add(new Setter() { Property = Shape.StrokeProperty, TargetName = UncheckedButton, Value = RadioButtonCheckMarkThemeColor });
+			checkedVisualState.Setters.Add(new Setter() { Property = Shape.StrokeProperty, TargetName = UncheckedButton, Value = radioButtonCheckMarkThemeColor });
```

Comment:
Could we use an `AppThemeBinding` here? So that it reacts to theme changes? 

Similar to this
https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Shell/BaseShellItem.cs#L427